api/pkg/apis/v1alpha1/managers/solution/solution-manager.go

```diff
@@ -268,16 +322,20 @@ func (s *SolutionManager) Reconcile(ctx context.Context, deployment model.Deploy
 	}
 
 	mergedState := MergeDeploymentStates(&currentState, desiredState)
-
 	var plan model.DeploymentPlan
 	plan, err = PlanForDeployment(deployment, mergedState)
 	if err != nil {
 		summary.SummaryMessage = "failed to plan for deployment: " + err.Error()
 		log.Errorf(" M (Solution): failed to plan for deployment: %+v", err)
-		s.saveSummary(iCtx, deployment, summary, namespace)
 		return summary, err
 	}
 
+	planBytes, _ := json.Marshal(plan)
```

Comment:
These look like leftover debugging statements. At minimum they should be logged as debug instead of info

Reply:
Switched them as debug logs.